tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
Sorry, see edit.  It was the trouble I'd already described (crashing, other tests failing) but there's more detail now.